packages/kolibri-common/components/SearchFiltersPanel/AccordionSelectGroup.vue

```diff
@@ -0,0 +1,468 @@
+          maxHeight: '256px',
+          overflowY: 'scroll',
+        }"
+      >
+        <template #content>
+          <KCheckbox
+            v-for="lang in languageOptionsList"
+            :key="'lang-' + lang.value"
+            :checked="isChecked('languages', lang)"
+            :disabled="lang.disabled || isEnabledButNotSelected('languages', lang)"
+            :label="lang.label"
+            @change="handleChange('languages', lang)"
+          />
+        </template>
+      </AccordionItem>
+    </AccordionContainer>
+
+    <AccordionContainer
+      v-if="contentLevelsList.length"
+      class="accordion-select"
+    >
+      <AccordionItem
+        :title="coreString('levelLabel')"
+        :headerAppearanceOverrides="accordionHeaderStyles(selectedLevel.value)"
+        :contentAppearanceOverrides="{
+          maxHeight: '256px',
+          overflowY: 'scroll',
+        }"
+      >
+        <template #content>
+          <KCheckbox
+            v-for="level in contentLevelsList"
+            :key="'level-' + level.value"
+            :checked="isChecked('grade_levels', level)"
+            :disabled="level.disabled || isEnabledButNotSelected('grade_levels', level)"
+            :label="level.label"
+            @change="handleChange('grade_levels', level)"
+          />
+        </template>
+      </AccordionItem>
+    </AccordionContainer>
+
+    <AccordionContainer
+      v-if="showChannels && channelOptionsList.length"
+      class="accordion-select"
+    >
+      <AccordionItem
+        :title="coreString('channelLabel')"
+        :headerAppearanceOverrides="accordionHeaderStyles(selectedChannel.value)"
+        :contentAppearanceOverrides="{
+          maxHeight: '256px',
+          overflowY: 'scroll',
+        }"
+      >
+        <template #content>
+          <KRadioButtonGroup>
+            <KRadioButton
+              v-for="channel in channelOptionsList"
+              :key="'channel-' + channel.value"
+              :buttonValue="channel.value"
+              :currentValue="selectedChannel['value'] || ''"
+              :label="channel.label"
+              @change="handleChange('channels', channel)"
+            />
+          </KRadioButtonGroup>
+        </template>
+      </AccordionItem>
+    </AccordionContainer>
+
+    <AccordionContainer
+      v-if="accessibilityOptionsList.length"
+      class="accordion-select"
+    >
+      <AccordionItem
+        :title="coreString('accessibility')"
+        :headerAppearanceOverrides="accordionHeaderStyles(selectedAccessibilityFilter.value)"
+        :contentAppearanceOverrides="{
+          maxHeight: '256px',
+          overflowY: 'scroll',
+        }"
+      >
+        <template #content>
+          <KCheckbox
+            v-for="a11y in accessibilityOptionsList"
+            :key="'a11y-' + a11y.value"
+            :checked="isChecked('accessibility_labels', a11y)"
+            :disabled="a11y.disabled || isEnabledButNotSelected('accessibility_labels', a11y)"
```

Comment:
I dont understand that much these conditions. We disable it if its enabled but not selected? I found it a little bit confusing :sweat_smile: